src/libslic3r/PrintConfig.cpp

```diff
@@ -1786,6 +1786,12 @@ void PrintConfigDef::init_fff_params()
     def->min = 0;
     def->set_default_value(new ConfigOptionFloats { 1.75 });
 
+    def = this->add("pellet_flow_coefficient", coFloats);
+    def->label = L("Pellet flow coefficient");
+    def->tooltip = L("Pellet flow coefficient is emperically derived and allows for volume calculation for pellet printer");
```

Comment:
Could you add more detailed descriptions here to help users understand how to calculate or set the parameter for their printer? Specifically, it would be helpful to include the relationship or formula that links this parameter to the filament diameter.

Reply:
Okay. I will add some helpful comments here and make it more coherent.

Reply:
Please take a look at the explanation. Hopefully this is clear enough.